android/app/src/googleplay/java/org/fossasia/openevent/fragments/MapsFragment.java

```diff
@@ -124,7 +118,29 @@ private void showLocationsOnMap() {
             builder.include(marker.getPosition());
         }
 
-        handleSearchEvents();
+        LayoutInflater inflator = (LayoutInflater) getActivity()
```

Comment:
@iamareebjamal  inflater is argument in onCreateView function. This is in showLocationsOnMap(). 
I think you forgot to expand the file, and took this to be in onCreateView.